winpr/libwinpr/clipboard/posix.c

```diff
@@ -439,31 +439,154 @@ static BOOL process_file_name(wClipboard* clipboard, const char* local_name, wAr
 	return result;
 }
 
-static BOOL process_uri(wClipboard* clipboard, const char* uri, size_t uri_len)
+bool is_dos_driver(const char* path)
 {
-	const char prefix[] = "file://";
-	BOOL result = FALSE;
-	char* name = NULL;
-	const size_t prefixLen = strnlen(prefix, sizeof(prefix));
-
-	WINPR_ASSERT(clipboard);
+	if ((path[1] == ':' || path[1] == '|') &&
+	    ((path[0] >= 'A' && path[0] <= 'Z') || (path[0] >= 'a' && path[0] <= 'z')))
+		return true;
+	return false;
+}
 
+char* parse_uri_to_local_file(const char* uri, size_t uri_len)
```

Comment:
should be `static` as only used inside this module.
to export for the unit test add something like:
```
#if !defined(BUILD_TESTING)
static
#endif
```